extensions/default/src/utils/index.ts

```diff
@@ -1 +1,2 @@
 export { addIcon } from './addIcon';
+export { Toolbox } from './Toolbox';
```

Comment:
Could we consistently use export * from './Toolbox'
so that it becomes easy to add new exports.  That also means any types defined will get exported.